data/transport missions.txt

```diff
@@ -1951,3 +1951,244 @@ ship "Science Drone"
+	source
+		near "Pherkad" 2 4
+		government "Free Worlds"
+	stopover "Cornucopia"
+	stopover "Solace"
+	stopover "Winter"
+	cargo "spaceport turrets" 75
+	to offer
+		random < 20
+		"combat rating" > 150
+	
+	on offer
+		conversation
+			`A captain in a Free Worlds uniform enters the spaceport and turns on a megaphone. "Attention! The Free Worlds are in need of a captain able to carry <cargo> to <stopovers> for fortifying the spaceports nearest to the pirate system of Men. Free Worlds ships are currently preoccupied with fighting off a pirate incursion into those systems, and we are unable to spare enough ships for the job. Our hope is that these defenses will deter any future incursions. Payment will be <payment> for the first captain who accepts." The spaceport falls quiet as captains contemplate this offer.`
+			choice
+				`	"I'll do it."`
+				`	(Ignore.)`
+					decline
+			
+			`	The Free Worlds captain thanks you for offering your help. Spaceport workers load the turrets onto your ship, and the Free Worlds captain tells you to return to <origin> once you have dropped off the turrets in order to pay you.`
+				accept
+	
+	npc
+		government "Pirate"
+		personality staying harvests plunders
+		system "Kappa Centauri"
+		fleet "Large Southern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Kappa Centauri"
+		fleet "Large Southern Pirates"
+	npc
+		government "Pirate"
+		personality staying harvests plunders
+		system "Yed Prior"
+		fleet "Large Southern Pirates"
+		fleet "Small Southern Pirates" 2
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Yed Prior"
+		fleet "Large Southern Pirates"
+	npc
+		government "Pirate"
+		personality staying harvests plunders
+		system "Pherkad"
+		fleet "Small Southern Pirates" 3
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Kappa Centauri"
+		fleet "Large Free Worlds"
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Yed Prior"
+		fleet "Large Free Worlds"
+	npc
+		government "Free Worlds"
+		personality staying heroic
+		system "Pherkad"
+		fleet "Large Free Worlds"
+	on stopover
+		dialog `The last of the defenses are quickly unloaded from your ship, put into place, and activated. One of the pirate ships attacking the system attempts to follow you down to the surface, but the fire coming from the newly installed spaceport turrets forces the pirate to turn away and flee for orbit. Time to return to <destination>.`
+	on complete
+		payment
+		payment 5000
+		dialog `The Free Worlds captain thanks you for helping to fortify the frontier worlds, and pays you the promised <payment>.`
+
+
+
+mission "Core Blockade"
+	minor
+	name `Bad Harvest`
+	description `The pirate world of <destination> is experiencing a famine. Warlords of the planet have threatened the Syndicate to give them food, and the Syndicate has agreed. Deliver <cargo> to the world for <payment>.`
```

Comment:
Maybe we can put something like `to their demands` after `has agreed`; it sounds a bit weird alone.